src/ctl_context.rs

```diff
@@ -113,7 +127,9 @@ impl Builder {
                                                         ptr::null_mut(),
                                                         &mut size);
             if res == winapi::FALSE {
-                return Err(io::Error::last_os_error());
+                let err = io::Error::last_os_error();
+                debug!("crypt and sign error: {:?}", err);
```

Comment:
Oh I was just using this for debugging, would you prefer if all debugging statements were removed? I figured I'd leave them in to see if they're helpful at some later date.